lib/node_modules/@stdlib/stats/base/dists/triangular/logpdf/README.md

```diff
@@ -145,10 +145,10 @@ var y;
 var i;
 
 for ( i = 0; i < 25; i++ ) {
-    x = randu() * 30.0;
-    a = randu() * 10.0;
-    b = a + (randu() * 40.0);
-    c = a + ((b-a) * randu());
+    x = uniform( 0.0, 30.0 );
+    a = uniform( 0.0, 10.0 );
+    b = a + ( uniform( 0.0, 40.0 ) );
```

Comment:
Here and elsewhere in this PR, we want to avoid adding several numbers together and prefer to just draw from the appropriate distribution. This can be rewritten more succinctly as

```suggestion
    b = uniform( a, a + 40.0 );
```